spec/Section 5 -- Validation.md

```diff
@@ -1133,7 +1133,7 @@ fragment catInDogFragmentInvalid on Dog {
 ##### Abstract Spreads in Object Scope
 
 In scope of an object type, unions or interface spreads can be used
-if the object type implements the interface or is a member of the union.
+if the type implements the interface or is a member of the union.
```

Comment:
technically the prior text was accurate (this is about object scope) but I see why you want to remove any chance for misreading